android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -386,6 +386,21 @@ private void sendAdLoadedReactNativeEventForAd(final MaxNativeAd ad)
         // 1. AdInfo for publisher to be notified via `onAdLoaded`
 
         WritableMap nativeAdInfo = Arguments.createMap();
+
+        nativeAdInfo.putString( "title", ad.getTitle() );
```

Comment:
What happens in the RN layer if you set `"title"` to `null`? Is it the same as not setting it?

Reply:
(same comment on iOS)

Reply:
It is the same as not setting it.  In the RN layer, both become `undefiend`. It was verified on iOS and Android.

Reply:
Cool, then there's no point in checking for nullability here (we can develop this feature with less code) right?

Reply:
Ok, will update without checking null.